src/views/ShoppingList.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="todo-page orange lighten-5" id="shoppinglist-page-container">
+    <v-text-field
+      v-model="newItem"
+      @click:append.prevent="addShoppingListItem"
+      @keyup.enter.prevent="addShoppingListItem"
+      class="pa-2 orange lighten-5"
+      outlined
+      label="Add an item"
+      append-icon="mdi-pencil-plus-outline"
+      clearable
+      hide-details
+    >
+    </v-text-field>
+    <v-list
+      v-if="checkShoppingList.length"
+      class="pa-0 orange lighten-5"
+      two-line
+      flat
+    >
+      <!-- Each Todo in Todo list -->
+      <div
+        id="item-container"
+        class="orange lighten-4"
+        v-for="item in checkShoppingList"
+        :key="item.id"
+      >
+        <v-list-item
+          @click="putItemInCart(item.id)"
+          :class="{ 'light-green accent-1': item.inCart }"
+        >
+        
+          <!-- Todo tick box     -->
+          <v-list-item-action>
+            <v-checkbox
+              :input-value="item.inCart"
+              color="light-green darken-4"
+            >
+            </v-checkbox>
+          </v-list-item-action>
+          <!-- Todo tick box ends -->
+
+          <!-- Todo list text -->
+          <v-list-item-content>
+            <v-list-item-title
+              :class="{ 'text-decoration-line-through': item.inCart }"
+            >
+              {{ item.item }}
+            </v-list-item-title>
+
+          </v-list-item-content>
+          <!-- Todo list text ends -->
+
+          <!-- Todo Menu Ends -->
+        </v-list-item>
+        <!-- Border Between Todos -->
+        <v-divider :key="item.id"></v-divider>
+      </div>
+      <!-- Each Todo in Todo list ends-->
+    </v-list>
+  </div>
+</template>
+
+<script>
+import { DateTime } from "luxon";
```

Comment:
Could you explain luxon and why it was needed? I thought we were using moment.js?

Reply:
Yes, can we make it a point to do this in Slack?

Reply:
Absolutely. Whatever is easiest for you